UPGRADE-3.4.md

```diff
@@ -210,6 +210,8 @@ Process
  * The `Symfony\Component\Process\ProcessBuilder` class has been deprecated,
    use the `Symfony\Component\Process\Process` class directly instead.
 
+ * Passing a not existing working directory to the constructor of the `Symfony\Component\Process\Process` class is deprecated and will not be supported anymore in Symfony 4.0.
```

Comment:
Actually `Process::__construct()` allows `null` for `$cwd` (and fills the property using `getcwd()`), allowing to call `setWorkingDirectory()` later. Unless we want to make it non nullable, this should talk about `start()`, not the constructor. 
I propose:

> Calling `Process::start()` without setting a valid working directory (via `setWorkingDirectory()` or constructor) beforehand is deprecated and will throw an exception in 4.0.